model/riscv_vlen.sail

```diff
@@ -1,10 +1,10 @@
 register elen : bits(1)
 
-val get_elen : unit -> {|32, 64|} effect {rreg}
+val get_elen_pow : unit -> {|5, 6|} effect {rreg}
 
-function get_elen() = match elen {
-    0b0 => 32,
-    0b1 => 64
+function get_elen_pow() = match elen {
```

Comment:
How was this function used prior to this change?  Benefit?
(The question is for educational purposes,  not to request a change back.)

Reply:
Before opening this PR I further read the Vector spec and noticed that the definition of ELEN is "the maximum size in bits of a vector element that any operation can produce or consume". So SEW should not be larger than ELEN which I hadn't considered this condition before. So I added this judgement into the function of getting the SEW value (`get_sew_pow()`). And for the convenience of comparing the value of SEW and ELEN, I modified ELEN to be the power value because SEW also uses the power value.